sdcm/cluster.py

```diff
@@ -1797,7 +1797,19 @@ def scylla_setup(self, disks):
         """
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
-        self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {}'.format(devname, ','.join(disks)))
+        if self.parent_cluster.params.get('workaround_kernel_bug_for_iotune'):
+            # related issue: https://github.com/scylladb/scylla/issues/5181
+            # a known kernel bug will cause scylla_io_setup fails in executing iotune.
+            # The kernel bug doesn't occur all the time, so we can get some succeed gce instance.
+            # the config files are copied from a succeed gce instance (same instance type, same test).
+            self.remoter.run(
+                'sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} --no-io-setup'.format(devname, ','.join(disks)))
+            for conf in ['io.conf', 'io_properties.yaml']:
+                self.remoter.send_files(src=os.path.join('./configurations/', conf), dst='/tmp/')
```

Comment:
better to put this to `data_dir`, all the configurations are there, no need to create new dir

Reply:
It's not a new directory  ('./configurations'), currently some EaR config files are stored there, they are configurations of scylla (scylla.yaml)

`data_dir/` is mess, it contains many kinds of configuration files.

Reply:
right, I missed that